shows/430 - Creator of Wordle.md

```diff
@@ -0,0 +1,81 @@
+---
+number: 430
+title: Creator of Wordle Josh Wardle
+date: 1644408000914
+url: https://traffic.libsyn.com/syntax/Syntax_-_430.mp3
+---
+
+In this episode of Syntax, Wes and Scott talk with the creator of Wordle, Jsoh Wardle. What's the tech stack for Wordle? Does he care about the clones of Wordle? And how did selling Wordle work?
+
+## Sentry - Sponsor
+
+If you want to know what’s happening with your code, track errors and monitor performance with Sentry. Sentry’s Application Monitoring platform helps developers see performance issues, fix errors faster, and optimize their code health. Cut your time on error resolution from hours to minutes. It works with any language and integrates with dozens of other services. Syntax listeners new to Sentry can get two months for  free by visiting [Sentry.io](https://sentry.io) and using the coupon code TASTYTREAT during sign up.
+
+## Sanity - Sponsor
+
+Sanity.io is a real-time headless CMS with a fully customizable Content Studio built in React. Get a Sanity powered site up and running in minutes at [sanity.io/create](https://www.sanity.io/create). Get an awesome supercharged free developer plan on [sanity.io/syntax](https://www.sanity.io/syntax).
+
+## Freshbooks - Sponsor
+
+Get a 30 day free trial of Freshbooks at [[freshbooks.com/syntax](http://freshbooks.com/syntax)]([https://freshbooks.com/syntax](https://freshbooks.com/syntax))
+
+FreshBooks is made for non-accountants like you. It’s an intuitive, easy-to-use accounting and invoicing software that does it all—from sending and following up on invoices, to automatically capturing receipt data for expenses, to getting you paid twice as fast.
+
+This episode is brought to you by FreshBooks, the invoicing and accounting solution that’s built for owners—and their clients.
```

Comment:
take the below freshbooks part out- this is the verbal ad read part - doesn't need to be in the show notes

Reply:
Done!